src/components/Table/Table_copy.jsx

```diff
@@ -25,27 +25,6 @@ const {
 } = PropTypes;
 
 const Thead = createClass({
-	displayName: 'Table.Thead',
```

Comment:
did you mean to include this file `Table_copy.jsx`?

Reply:
Ugh. Deleted that now too. Thanks, Joe!

Reply:
Nope. Took it out. Thx.